tests/test_biologic_reader.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+from matplotlib import pyplot as plt
+from ixdat.measurements import Measurement
+
+test_data_dir = Path(
+    "../test_data/biologic_mpt_and_zilien_tsv"
```

Comment:
I think is a little vulnerable, since it requires you to be in the folder when you run the file. I think it is better to do the path relative to the script path

Reply:
I don't understand, isn't this relative to the script path as it is?

Reply:
Ah, nevermind, I see. Yes. Changed to 
test_data_dir = `Path(__file__) / "../test_data/biologic_mpt_and_zilien_tsv"`

Reply:
... and now to `Path(__file__).parent.parent.parent.parent / "test_data/biologic_mpt_and_zilien_tsv"`
since the code has been moved to the reader module in src/ixdat/readers.
This strikes me also as a bit fragile. Is there a good way to reference the project folder from within a project module?

Reply:
.. and changed back again. See https://github.com/ixdat/ixdat/pull/2#discussion_r554450371

Reply:
That's a good question. I don't really have a good answer, maybe you can import the toplevel ixdat module and ask for its __file__, but that is somehow actually the same, except that you have the "dots" at the import instead of at the path.

Reply:
... and back again, now that it works in the module. ;)